Makefile

```diff
@@ -6,6 +6,7 @@ summary:
 	@echo "zero-aws-eks-stack:"
 	@echo "- Repository URL: ${REPOSITORY}"
 	@echo "- To see your kubernetes clusters, run: 'kubectl config get-contexts'"
-	@echo "- To switch to a cluster, use the NAME from the previous command in 'kubectl config use-context NAME'"
+	@echo "- To switch to a cluster, use the following commands:"
+	@echo $(shell echo ${ENVIRONMENT} | grep production > /dev/null && echo "- for production use: kubectl config use-context $(shell kubectl config get-contexts -o name | grep ${NAME}-production)")
```

Comment:
A cleaner way to do this comparison might be something like:
`[[ ${ENVIRONMENT} =~ "production" ]] && echo "..."`